scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2628,6 +2768,7 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
           token.is[Subtype] || token.is[Supertype] || token.is[Viewbound]
       )
     def loop(mods: List[Mod]): List[Mod] = token match {
+      case InlineSoftModifier() => mods
```

Comment:
```suggestion
      case InlineSoftIdent() => mods
```
otherwise we are suggesting we are not doing anything about those modifiers.

Reply:
:+1: